src/Pages/FieldInfoPage.js

```diff
@@ -7,6 +7,7 @@ import {
   Snackbar,
   FormHelperText,
   Typography,
+  Button
```

Comment:
please remove the fetch button, it's not needed anymore as location filtering is triggered by change in selection in the drop down. (Its removed already from master but this PR adds it again)